launch/launcher_test.go

```diff
@@ -314,7 +314,7 @@ func testLauncher(t *testing.T, when spec.G, it spec.S) {
 				})
 
 				when("there are zero args", func() {
-					it("treats the command as a bash script", func() {
+					it("treats the command as a bash script on linux", func() {
```

Comment:
```suggestion
					it("treats the command as a bash script on linux or invokes with cmd on windows", func() {
```

Reply:
(having `on linux` in the description makes me expect to see an `h.SkipIf(runtime.GOOS == "windows")`)